x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/random_sampler_range_slider.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiFlexItem, EuiFormRow, EuiRange, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { isDefined } from '@kbn/ml-is-defined';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { useState } from 'react';
+import {
+  MIN_SAMPLER_PROBABILITY,
+  RANDOM_SAMPLER_PROBABILITIES,
+  RANDOM_SAMPLER_STEP,
+} from '../../../index_data_visualizer/constants/random_sampler';
+
+export const RandomSamplerRangeSlider = ({
+  samplingProbability,
+  setSamplingProbability,
+}: {
+  samplingProbability?: number | null;
+  setSamplingProbability?: (value: number | null) => void;
+}) => {
+  // Keep track of the input in sampling probability slider when mode is on - manual
+  // before 'Apply' is clicked
+  const [samplingProbabilityInput, setSamplingProbabilityInput] = useState(samplingProbability);
+  const isInvalidSamplingProbabilityInput =
+    !isDefined(samplingProbabilityInput) ||
+    isNaN(samplingProbabilityInput) ||
+    samplingProbabilityInput <= 0 ||
```

Comment:
Should the check here be `<= 0.001`? Currently you can enter a value less than this and see the red border on the input but the button is still enabled.

<img width="331" alt="image" src="https://user-images.githubusercontent.com/7405507/217801688-f0ef4836-37fc-4abf-aff7-a7702a872b0e.png">

Reply:
Thanks for catching that. Fixed here [`9dcd987` (#150389)](https://github.com/elastic/kibana/pull/150389/commits/9dcd98713e36d060d3325a9a669cba374e4a5717)